cacheops/cleaner.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from .redis import redis_client
+from .sharding import get_prefix
+
+logger = logging.getLogger(__name__)
+
+
+def clear_stale_cacheops_keys(chunk_size: int, min_conj_set_size: int, dry_run: bool = False):
+    """
+    Remove expired cache keys from invalidation sets.
+
+    Cacheops saves each DB resultset cache key in a "conj set" so it can delete it later if it
+    thinks it should be invalidated due to a saved record with matching values. But the resultset
+    caches time out after 30 minutes, and their cache keys live in those conj sets forever!
+
+    So conj sets for frequent queries on tables that aren't updated often end up containing
+    millions of already-expired cache keys and maybe a few thousand actually useful ones,
+    and block Redis for multiple - or many - seconds when cacheops finally decides
+    to invalidate them.
+
+    This function scans cacheops' conj keys for already-expired cache keys and removes them.
+    """
+    logger.info('Starting scan for large conj sets')
+    prefix = get_prefix(dbs=['default'])
+    for conjunction_key in redis_client.scan_iter(f'{prefix}conj:*', count=chunk_size):
```

Comment:
It's better to call it `conj_key` since this is what we have everywhere. Besides esthetic purposes it has a practical one too: imagine one searching through code for `conjuction` and for `conj`.